From ca62c620bceab5319999046cfa61dea61e3bbe2d Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Thu, 31 Mar 2022 20:18:04 +0300
Subject: [PATCH] [SW-77251] common: add packed macro

This patch adds packed macro which is required for including specs
file.

Change-Id: Icd74b969f99fa36af2e10440a5337c8e79dfd4ac
---

diff --git a/include/arcboot.h b/include/arcboot.h
index ed4a1ec..c4ca14c 100644
--- a/include/arcboot.h
+++ b/include/arcboot.h
@@ -20,6 +20,10 @@
 /* TODO: check whether below command puts ARC to sleep */
 #define wfe()	__asm__ __volatile__("sleep 0x11\n" : : )
 
+#ifndef __packed
+#define __packed	__attribute__((packed))
+#endif
+
 #ifndef __weak
 #define __weak		__attribute__((weak))
 #endif
